x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/expression/predicate/operator/arithmetic/Div.java

```diff
@@ -34,4 +34,9 @@ protected Div replaceChildren(Expression newLeft, Expression newRight) {
     public DataType dataType() {
         return DataTypeConverter.commonType(left().dataType(), right().dataType());
     }
+
+    @Override
+    public ArithmeticOperation inverse(Source source, Expression left, Expression right) {
```

Comment:
return `Mul` instead of `ArithmeticOperation`

Reply:
Fixed.